e2e/terraform/packer/ubuntu-jammy-amd64/setup.sh

```diff
@@ -27,7 +27,10 @@ sudo apt-get upgrade -y
 sudo apt-get install -y \
      software-properties-common \
      dnsmasq unzip tree redis-tools jq curl tmux awscli nfs-common \
-     apt-transport-https ca-certificates gnupg2
+     apt-transport-https ca-certificates gnupg2 golang-go
+
+# Install hc-install
+sudo go install github.com/hashicorp/hc-install/cmd/hc-install@latest
```

Comment:
We _could_ get hc-install from an official release: https://releases.hashicorp.com/hc-install rather than have to ship the toolchain. Not sure if they have a deb in the official repo.

Reply:
Nice, didn't know we had that